.github/workflows/build_test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '21'
+          java-version: '23'
```

Comment:
Latest Java version is 24 not 23, is there a reason not to use it?

Reply:
It's due to compatibility :) 
https://docs.gradle.org/current/userguide/compatibility.html

Right now, gradle is not compatible with java 24.